pokemon-showdown

```diff
@@ -22,6 +24,84 @@ try {
 	child_process.execSync('npm install --production', {stdio: 'inherit'});
 }
 
+// Check if the server is configured to use Go, and ensure the required
+// environment variables and dependencies are available if that is the case
+
+let config;
+try {
+	config = require('./config/config');
+} catch (e) {}
+
+if (config && config.golang) {
+	if (!process.env.GOPATH) {
```

Comment:
GOPATH is optional in newer versions of Golang. You may want to assume `$HOME/go` path if it's not set or run `go env GOPATH` to obtain its location.